src/components/Modal/Form.jsx

```diff
@@ -0,0 +1,87 @@
+// import React, { useState } from 'react';
+import Inputs from './Inputs';
+import styled from 'styled-components';
+
+function Form({ format, handleModal, isSignUp, onClickSignUp }) {
+  // const [values, setValues] = useState();
+  const setValues = (e, input) => {
+    console.log(input);
+    console.log(`${input.type}: ${e.target.value}`);
+  };
+  return (
+    <>
+      <TitleWrapper>
+        <Title>{format.text}</Title>
+        <CloseBtn onClick={handleModal}>&times;</CloseBtn>
+      </TitleWrapper>
+      <InputContainer>
+        <Inputs format={format} setValues={setValues} />
+        <LinkBox>
+          {isSignUp ? (
+            <>
+              회원이신가요?&nbsp;
+              <RightAlignedLink onClick={onClickSignUp}>
+                로그인
+              </RightAlignedLink>
+            </>
+          ) : (
+            <>
+              아직 회원이 아니신가요?&nbsp;
+              <RightAlignedLink onClick={onClickSignUp}>
+                회원가입
+              </RightAlignedLink>
+            </>
+          )}
+        </LinkBox>
+      </InputContainer>
+    </>
+  );
+}
+
+const TitleWrapper = styled.header`
+  display: flex;
+  align-items: center;
+  height: 50px;
+  color: #ffff;
+  background-color: ${({ theme }) => theme.main};
+`;
+
+const Title = styled.span`
+  margin-left: 25px;
+  font-size: 20px;
+  color: white;
+`;
+
+const CloseBtn = styled.button`
+  position: absolute;
+  right: 15px;
+  outline: none;
+  cursor: pointer;
+  border: 0;
+  font-size: 30px;
+  color: #ffff;
+`;
+
+const InputContainer = styled.div`
+  padding: 18px;
+  background: white;
+`;
+
+const LinkBox = styled.div`
+  margin-top: 7px;
+  margin-right: 9px;
+  text-align: right;
+  color: gray;
+`;
+
+const RightAlignedLink = styled.a`
```

Comment:
a 태그 사용하면 페이지가 새로고침 되지 않을까요? 공식문서 확인해보시고 아래처럼 사용해주세요
```suggestion
const RightAlignedLink = styled(Link)`
```

Reply:
https://gomgomkim.tistory.com/9
관련해서 글 읽고 블로그에도 글 남길 계획입니다. 감사합니다!